src/README.md

```diff
@@ -1,9 +1,14 @@
+<<<<<<< HEAD
+Collect data from StatsCan and other statistical agencies and convert them to clean CSV files.
+=======
 Collect data from StatsCan and other statistical agencies and convert to clean CSV files for use in a secure research area. Note that this code **does not** produce the actual SES index, but only produces a subset of the required cleaned datasets.
+>>>>>>> 9bc117ff62f1f313d9e0fcad773b4ea3379b731c
 
 
 ### Data
 
 1. [StatsCan census 2021](https://www12.statcan.gc.ca/census-recensement/2021/dp-pd/index-eng.cfm). It includes income, education, vocation, and family information. 
+  * A `background.txt` stores the information of the detail of StatsCan Census data.
```

Comment:
not sure where this is?

Reply:
After a discussion with Tim, we decided to move a long comment from R code file to a separate `background.txt` which is about how Census data is organized.

Reply:
is that the background.R file in the census folder? And do we also want this script in that folder, or do we even need the subfolders?

Reply:
This is background information. Including it in the R code would be too long and cumbersome. Previously, we put them in a Txt file, and later changed it to r. I am not sure what are the best ways to save that information.

Reply:
Yeah I was just confused because the README said `background.txt` but the file was `background.R`. It doesn't really matter -> if it's truly background that's needed in here, markdown is probably the way to go :) (github loves markdown!) 

Other question still stands -> do we need the subfolders? And if so should the code be in the subfolder too? Or can we just have a src folder, and then a background_info folder with a slightly better named `background.???` file.

Reply:
I agree. It seems to be a better solution. One markdown file for each data and all markdown files are saved in the background_info folder.

Reply:
Well, one markdown if it's needed, some may not need it! (I think census just had a lot of extra info)

Reply:
Sure. The comments were less organized in R code. I will clean them and put all background info in one markdown file.

Reply:
Done. Now we have one folder and one file for the background info.